### PR TITLE
Views

### PR DESCRIPTION
**Pull request by KayLemba**
In this feature update i did the following:

- Implement the design from the sneak peek wireframes.
- Use methods created in the previous branch

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>Here is a list of users</h1>
\ No newline at end of file
+<div class="list">
+  <% @users.each do |user| %>
+    <%= link_to user do %>
+      <%= render partial: "user", object: user %>
+    <% end %>
+  <% end %>
+</div>
```
Comment (ahmetbozaci) [inline app/views/users/index.html.erb]: ### OPTIONAL
- [x] To show all users you don't have to use each method

Example:
image

File: app/views/users/_user.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    <%= image_tag user.photo %>
+    </figure>
+  </div>
+  <div class="media-content notification">
+    <div class="content">
+      <h4 class="title is-4"><%= user.name %></h4>
+      <p class="has-text-right is-size-7"><%= t('Posts_Counter', count: user.Posts_Counter)%></p>
+    </div>
+  </div>
+</article>
```
Comment (ahmetbozaci) [inline app/views/users/_user.html.erb]: - [x] Style does not matter in this milestone. But at least change the layout like in the wireframe.
- [x] Also ı cant see the post counter number on this page. Please fix it.
image

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+<%= render @user %>
+<% if @user.bio.present? %>
+	<div class="mt-5 notification">
+	  <h5 class="title is-6">Bio</h5>
+	  <p class="is-6"><%= @user.bio %></p>
+	</div>
+<% end %>
+<%= render partial: "posts/post", collection: @posts %>
+<%= link_to "See all post", user_posts_path(@user.id), class: "button is-link is-small" %>
```
Comment (ahmetbozaci) [inline app/views/users/show.html.erb]: - [x] I should see the recent post method in the user profile. Please fix it.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @KayLemba  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

- [ ] Please change the style layout on all pages as in the wireframe. 

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

File: app/views/users/show.html.erb (hunk 3 above)
Comment (KayLemba) [reply app/views/users/show.html.erb]: Thanks You for  the review @ahmetbozaci

File: app/views/users/_user.html.erb (hunk 2 above)
Comment (KayLemba) [reply app/views/users/_user.html.erb]: done

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -1 +1,38 @@
-<h1>Here is a profile of a given user</h1>
\ No newline at end of file
+<div class="container">
+    <div class="card mb-3 border-info mb-3" style="max-width: 900px">
+      <div class="row g-0">
+        <div class="col-md-4">
+          <img src=<%@user.name%> class="img-fluid rounded-start">
```
Comment (Whoistolu) [inline app/views/users/show.html.erb:5]: - [x] [OPTIONAL]KIndly ensure you are displaying each user image, this will make your project look good

**Review by Whoistolu — APPROVED** [review]
### Hi @KayLemba 

Great job implementing the recommended changes
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 4 above)
Comment (KayLemba) [reply app/views/users/show.html.erb:5]: Thank you for a review i'll add the images @Whoistolu

**Pull request merged**